dashboard/db/schema.rb

```diff
@@ -2330,6 +2345,8 @@
   add_foreign_key "learning_goal_ai_evaluations", "learning_goals"
```

Comment:
curious to know more here with the FK's -- the second clause is the ID you are referencing?

Reply:
Yeah, and I think Rails automatically knows which column to use. For example in `add_foreign_key "lti_courses", "lti_deployments"` it knows to add the constraint to the `lti_deployment_id` column in the `lti_courses` table.